cfme/tests/configure/test_logs.py

```diff
@@ -130,20 +125,38 @@ def test_provider_log_level(appliance, provider, log_exists):
         initialEstimate: 1/4h
         casecomponent: Configuration
         testSteps:
-            1. Change log level to debug
+            1. Change log level to warn
             2. Refresh provider
-            3. Check logs contain debug level
+            3. Check logs do not contain info
             4. Reset level back
     """
     assert log_exists, "Log file {}.log doesn't exist".format(provider.log_name)
     log_level = appliance.server.advanced_settings['log']['level_{}'.format(provider.log_name)]
-    # set log level to debug
+    log = f'/var/www/miq/vmdb/log/{provider.log_name}.log'
+    # set log level to info
     wait_for(lambda: appliance.server.update_advanced_settings(
-        {'log': {'level_{}'.format(provider.log_name): 'debug'}}), timeout=300)
+        {'log': {'level_{}'.format(provider.log_name): 'info'}}), timeout=300)
+    lv_info = LogValidator(log, matched_patterns=['.*INFO.*'])
```

Comment:
@lina-nikiforova can you make use of the new contextmanager for the `LogValidator`? https://github.com/ManageIQ/integration_tests/blob/d76fe2e735c0b45c63985bed87644e482dc98e42/cfme/utils/log_validator.py#L132-L136

Reply:
@john-dupuy I tried that but it gave me some error `AttributeError: __enter__`